src/test/scala/aserralle/akka/stream/kcl/KinesisWorkerSourceSourceSpec.scala

```diff
@@ -285,6 +388,7 @@ class KinesisWorkerSourceSourceSpec
               KinesisWorkerCheckpointSettings(maxBatchSize = 100,
                                               maxBatchWait = 500.millis))
         )
+        .withAttributes(Attributes.inputBuffer(initial = 200, max = 256))
```

Comment:
Is this really needed? Your test reproduced the problem with just 25 messages.

Reply:
nope, i will remove it, thanks for catching